### ISSUE TITLE
Incorrect SOMessageHelper allocation

### ISSUE THREAD

**Issue by smartfuse** [CamHenlin/nodeprivatemessageskit#1]
You can't just create a SOMessageHelper, you have to use a sharedInstance and then it will work!

Change
`var messageHelper = $.SOMessageHelper('alloc')('init');`
to
`var messageHelper = $.SOMessageHelper('weakSharedInstance')`

**Comment by ZekeSnider** [comment]
This is correct, I was able to send messages using `SOMessageHelper` with `weakSharedInstance`. Still trying to figure out how to send group messages though.
